src/components/__tests__/RegistryAdmin.test.tsx

```diff
@@ -88,40 +97,60 @@ describe('RegistryAdmin, with access token', () => {
 
     expect(libraryAccordions.length).toEqual(15);
   });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
 });
 
 describe('RegistryAdmin, with no access token, but with refresh token', () => {
-  it('tries to refresh the accessToken', () => {
+  it('tries to refresh the accessToken', async () => {
     const realGet = Cookies.get;
 
     Cookies.get = jest.fn().mockImplementation(() => 'mockRefreshToken');
     (global as any).fetch = jest.fn().mockReturnValueOnce(
       Promise.resolve({
+        ok: true,
         status: 200,
         json: () => Promise.resolve({ access_token: 'mockAccessToken' }),
       })
     );
 
-    renderRegistryAdminWithContext('');
+    await act(() => renderRegistryAdminWithContext(''));
 
-    expect(fetch).toHaveBeenCalledWith(
-      'https://qa-libraryregistry.librarysimplified.org/admin/refresh_token',
-      { headers: { Authorization: 'Bearer mockRefreshToken' }, method: 'POST' }
-    );
+    expect(fetch).toHaveBeenCalledWith(process.env.REFRESH, {
+      headers: { Authorization: 'Bearer mockRefreshToken' },
+      method: 'POST',
+    });
 
     Cookies.get = realGet;
```

Comment:
Hm I don't quite understand this and maybe I missed it in a previous PR. Is it getting the cookies here for an assertion that we want to make?

Reply:
Yes, thank you.